qa/entitlements/src/main/java/org/elasticsearch/test/entitlements/EntitlementsCheckPlugin.java

```diff
@@ -42,6 +41,6 @@ public List<RestHandler> getRestHandlers(
         final Supplier<DiscoveryNodes> nodesInCluster,
         Predicate<NodeFeature> clusterSupportsFeature
     ) {
-        return Collections.singletonList(new RestEntitlementsCheckSystemExitAction());
+        return List.of(new RestEntitlementsCheckSystemExitAction(), new RestEntitlementsCheckClassLoaderAction());
```

Comment:
I'd forgotten how cool this part is. 😄 

This is of our white-hat hacker testing. This unruly plugin lets users attempt everything they're not supposed to be allowed to do.

Reply:
Yes :) 
And we can have another plugin with permissions/with different policies. 
I'm going to add a non-modular plugin as well next, to test that class -> plugin resolution works with both kinds. It's quite easy and very realistic.